common/twmap.hh

```diff
@@ -94,6 +96,18 @@ public:
     return count;
   }
   int sum(const std::string& s, const TWStatsBuf& vec) { return 0; }
+  void dump(std::ostream& os) {
```

Comment:
you should add & user 'override' keyword with members that override methods, this way the compiler can tell you if it changes in the future.

Reply:
yep good point